### ISSUE TITLE
Docs installations

### ISSUE THREAD

**Issue by lxgnzlz05** [Erythrocyte3803/PHP-Yggdrasil-Server#1]
Hii, Can you provide Docs? i dont know what database use and.. can you provide the skin site code and the login site code? Thanks!

**Comment by candied-apple** [comment]
Yeah can u do docs pls and registration page docs i need this for my launcher
